docker/package/tezos_baking_wizard.py

```diff
@@ -210,166 +245,121 @@ def pprint_options(self):
+    help="Starting the service will make it available just for this session, great\n"
+    "if you want to experiment. Enabling it will make it start on every boot.",
+    options=systemd_enable,
+    validator=Validator(enum_range_validator(systemd_enable)),
+)
+
+snapshot_mode_query = Step(
+    id="snapshot",
+    prompt="The Tezos node can take a significant time to bootstrap from scratch.\n"
+    "Bootstrapping from a snapshot is suggested instead.\n"
+    "How would you like to proceed?",
+    help="A fully-synced local tezos-node is required for running a baking instance.\n"
+    "By default, service with tezos-node will start to bootstrap from scratch,\n"
+    "which will take a significant amount of time.\nIn order to avoid this, we suggest "
+    "bootstrapping from a snapshot instead.\n\n"
+    "Snapshots can be downloaded from the following websites:\n"
+    "Tezos Giganode Snapshots - https://snapshots-tezos.giganode.io/ \n"
+    "XTZ-Shots - https://xtz-shots.io/ \n\n"
+    "We recommend to use rolling snapshots. This is the smallest and the fastest mode\n"
+    "that is sufficient for baking (you can read more about other tezos-node history modes here:\n"
+    "https://tezos.gitlab.io/user/history_modes.html#history-modes ).\n"
+    "All commands within the service are run under the 'tezos' user.",
+    options=snapshot_import_modes,
+    validator=Validator(enum_range_validator(snapshot_import_modes)),
+)
+
+snapshot_file_query = Step(
+    id="snapshot_file",
+    prompt="Provide the path to the tezos-node snapshot file.",
+    help="You have indicated wanting to import the snapshot from a file.\n"
+    "You can download the snapshot yourself e.g. from XTZ-Shots or Tezos Giganode Snapshots.",
+    validator=Validator(filepath_validator),
+)
+
+snapshot_url_query = Step(
+    id="snapshot_url",
+    prompt="Provide the url of the tezos-node snapshot file.",
+    help="You have indicated wanting to import the snapshot from a custom url.\n"
+    "You can use e.g. links to XTZ-Shots or Tezos Giganode Snapshots resources.",
+)
+
+key_mode_query = Step(
+    id="key_import_mode",
+    prompt="How do you want to import the baker key?",
+    help="To register the baker, its secret key needs to be imported to the data "
+    "directory first.\nBy default tezos-baking-<network>.service will use the 'baker' "
+    "alias\nfor the key that will be used for baking and endorsing.\n"
+    "If you want to test baking with a faucet file, make sure you have chosen a test network like "
+    + list(networks.keys())[1],
+    options=key_import_modes,
+    validator=Validator(enum_range_validator(key_import_modes)),
+)
+
+secret_key_query = Step(
+    id="secret_key",
+    prompt="Provide either the unencrypted or password-encrypted secret key for your address.",
+    help="The format is 'unencrypted:edsk...' for the unencrypted key, or 'encrypted:edesk...'"
+    "for the encrypted key.",
+    validator=Validator(required_field_validator),
```

Comment:
Let's validate that the provided secret key matches the expected form: `{encrypted, unencrypted}:<base58 encoded string with length 54>`